page/cyber-security.php

```diff
@@ -47,46 +48,19 @@
             <input type="text" class="search" name="search" placeholder="Quick Search">
         </div>
         <div class="col-md-9 row">
-            <div class="col-md-3 mb-4">
-                <div class="card">
-                    <img class="card-img-top" src="img/senjata/SPR_4.jpeg">
-                    <div class="card-body font-weight-bold">
-                        <p>SPR-4</p>
-                    </div>
-                </div>
-            </div>
-            <div class="col-md-3 mb-4">
-                <div class="card">
-                    <img class="card-img-top" src="img/senjata/SPR_4.jpeg">
-                    <div class="card-body font-weight-bold">
-                        <p>SPR-4</p>
-                    </div>
-                </div>
-            </div>
-            <div class="col-md-3 mb-4">
-                <div class="card">
-                    <img class="card-img-top" src="img/senjata/SPR_4.jpeg">
-                    <div class="card-body font-weight-bold">
-                        <p>SPR-4</p>
-                    </div>
-                </div>
-            </div>
-            <div class="col-md-3 mb-4">
-                <div class="card">
-                    <img class="card-img-top" src="img/senjata/SPR_4.jpeg">
-                    <div class="card-body font-weight-bold">
-                        <p>SPR-4</p>
-                    </div>
-                </div>
-            </div>
-            <div class="col-md-3 mb-4">
-                <div class="card">
-                    <img class="card-img-top" src="img/senjata/SPR_4.jpeg">
-                    <div class="card-body font-weight-bold">
-                        <p>SPR-4</p>
-                    </div>
-                </div>
-            </div>
+            <?php
+                $select = mysqli_query($connection, "select * from produk where jenis_produk='Senjata'");
+                while($data = mysqli_fetch_array($select)){
```

Comment:
Codacy found an issue: [The use of function mysqli_fetch_array() is discouraged](https://app.codacy.com/manual/anamnafiul99/pindad-clone/pullRequest?prid=5762293)